features/step_definitions/element_explicit_waiting_steps.rb

```diff
@@ -135,3 +105,16 @@
 
   expect(@duration).to be_between(0.5, 0.7)
 end
+
+Then('I am not made to wait to check a nonexistent element for invisibility') do
+  start = Time.new
+  @test_site.home.wait_until_nonexistent_element_invisible(wait: 10)
+
+  expect(Time.new - start).to be < 1
+end
+
+Then('an error is thrown when waiting for an element in a vanishing section') do
```

Comment:
an **Element Not Found** error is thrown when waiting for an element in a vanishing section